### PR TITLE
Workflow: Add "Help" and "How It Works"

### PR DESCRIPTION
**Pull request by dvandra**
Adding "Help" and "How It Works" in Skydive Workflow in-order to make it more generic and user-friendly.

### CODE REVIEW

File: statics/workflows/flow-validation.yaml [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
 UUID: "1c79a9d2-6fcb-4e10-7d40-78819b07781c"
 Name: "FlowValidation"
-Description: "Check Flow-Forwarding form Checkpoint-1 to Checkpoint-2"
+Abstract: "Check Flow-Forwarding form Checkpoint-1 to Checkpoint-2"
+Description:
+  Help:
+    1: Use - To validate packet-flows on one/two/between two checkpoints
+    2: To validate on just one checkpoint select it as Checckpoint-1
+    3: To validate on two checkpoints select it as Checkpoint-1 and Checkpoint-2
+    4: To validate on all nodes between Checkpoint-1 and Checkpoint-2, Select the checkbox goiven below Checkpoint-2
```
Comment (safchain) [inline statics/workflows/flow-validation.yaml]: s/goiven/given/

Comment (dvandra) [reply statics/workflows/flow-validation.yaml]: ohh!

Comment (dvandra) [reply statics/workflows/flow-validation.yaml]: Done, Thank You!

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Pull request merged**